jobs/schedulers.go

```diff
@@ -148,7 +147,7 @@ func (schedulers *Schedulers) setNextRunTime(cfg *model.Config, idx int, now tim
 
 	if !pendingJobs {
 		if pj, err := schedulers.jobs.CheckForPendingJobsByType(scheduler.JobType()); err != nil {
-			mlog.Error("Failed to set next job run time: " + err.Error())
+			mlog.Error("Failed to set next job run time", mlog.Err(err))
```

Comment:
GIven that `err` is of `*model.AppError` this should be changed to  `mlog.Error(err.Error())`

Reply:
No @marianunez this is *model.AppError, which fullfil the error interface, so we are trying to convert everything to structured logging, in this case, the `mlog.Err` is the correct approach.

Reply:
Gotcha! 👌